### DISCUSSION TITLE
Feature request: `dvc plots diff` output for all figures in md file

### DISCUSSION THREAD

**Discussion by aguschin** [treeverse/dvc#6165] [New Features & Ideas]
Now if we want to append images to comment for commit, we need to specify each image one by one:

```
# in CI:
echo "## Plots" >> report.md
dvc plots diff --targets runs/train/metrics/val_cls_loss.tsv --show-vega $DEFAULT_BRANCH > vega.json
vl2png vega.json | cml-publish --md >> report.md
dvc plots diff --targets runs/train/metrics/val_box_loss.tsv --show-vega $DEFAULT_BRANCH > vega.json
vl2png vega.json | cml-publish --md >> report.md

cml-send-comment report.md
```

It would be great to have some magic to send all plots (dvc plots diff) to the comment of the commit. If I have 10+ of these plots, I will end up either copy-pasting this, or writing some scripts which will find all plots and do this for me. Something like this:

```
echo "## Plots" >> report.md
dvc plots diff --show-vega-to-md $DEFAULT_BRANCH >> report.md
```

This may not belong to dvc directly and more about dvc+cml integration, I guess, but nevertheless the use case still have a place. Or maybe I'm too lazy and I need to write for loop in bash enumerating all possible `--targets` :)

**Comment by dberenbaum** [comment]
Seems like there are a lot of ways to solve this. What you suggested makes sense: a `--show-md` (don't think it's necessary to mention `vega`) option that acts like the html version of `dvc plots diff` but generates png files for each plot and appends them to a markdown file.

I also agree that it's more about cml integration, so we probably need to get a sense of how common this pattern is and how important a feature it is for cml.
